### DISCUSSION TITLE
Allure stops reporting after upgrade of bamboo

### DISCUSSION THREAD

**Discussion by fixxxera** [allure-framework/allure2#2158] [Questions & Support] [answered]
Bamboo upgraded from 7.1.4 > 8.2.9 > 9.2.5 LTS. 
All plugins are updated. Tasks that have allure reporting stopped reporting. Previous reports are visible. Any ideas? What more info will be helpful?

**Comment by eroshenkoam** [comment]
We're aware of the issue with the Allure reports after updating Bamboo. We're looking into it and aim to fix it by the end of this week. Thank you for your feedback!

Best wishes! 😊

**Comment by eroshenkoam** [comment, marked as answer]
We've fixed the plugin for Bamboo. Please check version 1.17.0. Let us know if everything works well, or if you encounter any issues.
https://marketplace.atlassian.com/apps/1217177/allure-report-for-bamboo/version-history#b1000160
